scrapy_autoextract/providers.py

```diff
@@ -55,11 +55,17 @@ def _stop_if_account_disabled(exception: Exception, crawler: Crawler):
 
 
 def get_concurrent_requests_per_domain(settings: Settings):
-    """Return the configured concurrent request per domain from settings"""
-    concurrency = settings.getint("CONCURRENT_REQUESTS_PER_DOMAIN", 8)
+    """Return the configured AutoExtract concurrent request per domain from settings"""
+    limit_name = "AUTOEXTRACT_CONCURRENT_REQUESTS_PER_DOMAIN"
+    concurrency = settings.getint(limit_name, -1)
+    # If no AutoExtract-specific limit is provided - use the default one
+    if concurrency is -1:
+        limit_name = "CONCURRENT_REQUESTS_PER_DOMAIN"
+        concurrency = settings.getint(limit_name, 8)
     if concurrency < 1:
-        raise ValueError("Invalid 'CONCURRENT_REQUESTS_PER_DOMAIN' "
+        raise ValueError(f"Invalid '{limit_name}' "
                          f"value: {concurrency}")
+    logger.info(f"Setting `concurrent requests per domain` limit to {concurrency}.")
```

Comment:
It'd be better not to log this line here - this function gets the concurrent requests to be used, not sets it. I'm also not sure it should be an INFO log message - what do you think about changing it to DEBUG? 

Option value is logged anyways.